packages/fx-core/src/plugins/resource/function/resources/errors.ts

```diff
@@ -132,6 +135,18 @@ export class UnzipError extends FunctionPluginError {
   }
 }
 
+// TODO: help link for the error
+export class RegisterResourceProviderError extends FunctionPluginError {
+  constructor() {
+    super(
+      ErrorType.User,
+      "RegisterResourceProviderError",
+      "Failed to register required resource provider for function app.",
+      [tips.registerRequiredRP, tips.checkLog]
```

Comment:
Nit: due to the access control, better to mention that user should have enough rights